src/Tests.props

```diff
@@ -6,6 +6,8 @@
     <SingleTestPlatform Condition="'$(SingleTestPlatform)' == ''">false</SingleTestPlatform>
     <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
     <IsPackable>false</IsPackable>
+    <Nullable>enable</Nullable>
```

Comment:
Please remove this property from the test projects. Or, maybe NRT can be enabled in the `Directory.Build.targets` unconditionally and removed from other places?

Reply:
> Please remove this property from the test projects. 

Good catch; done.

> Or, maybe NRT can be enabled in the `Directory.Build.targets` unconditionally and removed from other places?

Maybe; let's get the bulk of the changes merged first.